apps/wallet/src/ui/app/components/active-coins-card/index.tsx

```diff
@@ -48,21 +49,22 @@ export function ActiveCoinsCard({
     activeCoinType: string;
     showActiveCoin?: boolean;
 }) {
-    const aggregateBalances = useAppSelector(accountAggregateBalancesSelector);
+    const primaryAddress = useAppSelector(({ account }) => account.address);
 
-    const coins = useMemo(
-        () =>
-            Object.entries(aggregateBalances).map((aType) => {
-                return {
-                    coinType: aType[0],
-                    balance: aType[1],
-                };
-            }),
-        [aggregateBalances]
+    // TODO: remove and use useGetAllBalances hook, once it's merged
```

Comment:
We should get into the habit of stacking PRs to avoid this issue

Reply:
This is a stacked PR from send max PR, the problem is this PR is far ahead that it word be difficult to rebase from coin management PRs.